### PR TITLE
Allow multiple images to be added at once.

### PR DESCRIPTION
**Pull request by Prof-Butts**
This helps when loading animations (which is otherwise a very tedious operation).

### CODE REVIEW

File: XwaDatEditor/XwaDatEditor/MainWindow.xaml.cs [hunk 1]
```diff
@@ -449,12 +444,15 @@ private void AddImage_Click(object sender, RoutedEventArgs e)
...
+                        {
+                            var image = DatImage.FromFile(group.GroupId, (short)(imageId + i), dialog.FileNames[i]);
 
-                        group.Images.Add(image);
+                            group.Images.Add(image);
 
-                        disp(() => this.DatFile = this.DatFile);
-                        disp(() => this.ImagesList.SelectedIndex = this.ImagesList.Items.Count - 1);
+                            disp(() => this.DatFile = this.DatFile);
+                            disp(() => this.ImagesList.SelectedIndex = this.ImagesList.Items.Count - 1);
```
Comment (JeremyAnsel) [inline XwaDatEditor/XwaDatEditor/MainWindow.xaml.cs]: Please move these 2 `disp` lines outside the `for` loop. It is not needed to update the UI after each image is added but only after all new images are added.

**Comment by JeremyAnsel** [comment]
One request. Except that, LGTM.
Thanks for the contribution.

File: XwaDatEditor/XwaDatEditor/MainWindow.xaml.cs (hunk 1 above)
Comment (Prof-Butts) [reply XwaDatEditor/XwaDatEditor/MainWindow.xaml.cs]: Done!

**Comment by Prof-Butts** [comment]
No problem. The PR has been updated.

**Pull request merged**